wesnoth-ei.po

```diff
@@ -4572,15 +4619,13 @@ msgstr ""
 
 #. [objective]: condition=win
 #: data/campaigns/Eastern_Invasion/scenarios/09_Castle_in_the_Ice.cfg:442
-#, fuzzy
-#| msgid "Come, into the castle."
 msgid "Move any unit to the abandoned castle."
-msgstr "Pojďte do hradu."
+msgstr "Přesuň jakoukoli jednotku do opuštěného hradu."
 
 #. [objective]: condition=win
 #: data/campaigns/Eastern_Invasion/scenarios/09_Castle_in_the_Ice.cfg:447
 msgid "Move Grug next to an enemy ogre."
-msgstr ""
+msgstr "Přesuň Gruga vedle nepřátelského obra"
```

Comment:
```suggestion
msgstr "Přesuň Gruga vedle nepřátelského obra."
```

Reply:
Tady jde o to, že všechny cíle scénaře (tedy výhry i prohry) se standartně dělají bez tečky. Takže tohle je špatně v originále. Nebo to všude změnit a dávat tečky. To by se mi líbilo víc, jsou to přece jenom věty.

Reply:
Jasně, to jsem přeťal. Koukám na [psaní výčtů](https://prirucka.ujc.cas.cz/?id=870), mně by se to líbilo malým na začátku a bez tečky na konci, ale zatím bych to nechal tak, jak to je.

Reply:
Ne, nepřeťal, teď to vidím - srovnal jsem to s předchozí podmínkou vítězství (obě mají v předloze tečku, tvůj překlad jen v první podmínce). Protože trváš na beztečkovosti u druhé podmínky, odebírám tečku i z první.

Reply:
Aha, to jsem přehlédl.